ansible/roles/lvm-partition/tasks/main.yml

```diff
@@ -1,23 +1,24 @@
 ---
 # Setup LVM partition
 
-- name: storage | create logical volume
+- name: storage | mount | create logical volume
   lvol:
-    vg: "{{ vgname }}"
-    lv: "{{ lvname }}"
-    size: "{{ lvsize }}"
-    opts: "{{ lvopts }}"
+    vg: "{{ lvm_vgname }}"
+    lv: "{{ lvm_lvname }}"
+    size: "{{ lvm_lvsize }}"
+    opts: "{{ lvm_lvopts }}"
```

Comment:
While testing I realized `lvol` `opts` is a 2.0+ feature. Are you using development ansible or a beta version @kennethgillen @manics ?